cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/ReadOnlyKafkaSampleStore.java

```diff
@@ -36,7 +36,6 @@ public void configure(Map<String, ?> config) {
     for (int i = 0; i < numProcessingThreads; i++) {
       _consumers.add(createSampleStoreConsumer(config, CONSUMER_CLIENT_ID_PREFIX));
     }
-    _loadingProgress = -1.0;
```

Comment:
Wouldn't this change cause `_loadingProgress` to be unassigned? Should we instead have `_loadingProgress = LOADING_PROGRESS;` here?

Note: we can also delete the function `evictSamplesBefore(long timestamp)`, but feel free to ignore this, because it is not introduced in this PR.